src/bin/bat/clap_app.rs

```diff
@@ -257,6 +257,15 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      an interactive terminal is detected. Possible values: *auto*, never, always.",
                 ),
         )
+        .arg(
+            Arg::with_name("always-decorations")
+                .short("f")
+                .alias("always-decor")
+                .overrides_with("always-decorations")
+                .hidden(true)
+                .hidden_short_help(true)
+                .help("Alias for '--decorations=always --color=always'")
```

Comment:
Can we please use a proper sentence here like for the rest of the command line options? Maybe we could also add:

"This is useful if the output of bat is piped to another program, but you want to keep the colorization/decorations."